src/xcode/ENA/ENA/Source/HTTPClientCore/Service/Services/_Service+Default.swift

```diff
@@ -57,12 +57,33 @@ extension Service {
 			completion(.success(receiveModel))
 			return
 		}
-		
+
+		// To check if we want to retry the load call, we hook in the completion call.
+		let retryingCompletion: ((Result<R.Receive.ReceiveModel, ServiceError<R.CustomError>>) -> Void) = { result in
+			switch result {
+
+			case .success(let model):
+				completion(.success(model))
+			case .failure(let error):
+				// For any failures we check if the resource has a retry flag and has some retries left and if so, we call load() recursivly.
```

Comment:
I guess there is no retry flag anymore?

Reply:
Modified the comment to point better out, what we expect here.